redhat-developer/developers.redhat.com
# [RHDX-145] Put Adobe scripts in html.html.twig

In d7b673b346a301d6da812816e61ab0b60d89aa4a, we moved some scripts that
lived in html.html.twig into Drupal libraries to make the html.html.twig
file cleaner/smaller. This had the unintended consequence of loading
some scripts too late or out of order. The sendCustomEvent() function,
for example, wasn't available. To resolve this, I will revert the
changes for at least the Adobe DDO integration.

### JIRA Issue Link
* https://projects.engineering.redhat.com/browse/RHDX-145

### Verification Process

* Go to a page that loads the `rhdp2` theme, for example the homepage
* Open the inspector and `console.log(digitalData)`
* You should see something like this:
```
{
  "cart": {},
  "event": [
    {
      "eventInfo": {
        "eventName": "user data",
        "eventAction": "available",
        "user": [
          {
            "profile": [
              {
                "profileInfo": {
                  "accountID": "",
                  "daysSinceLastPurchase": "",
                  "daysSinceRegistration": "",
                  "eloquaGUID": "271467ab-a3be-4c0e-b60b-905daa84c28c",
                  "keyCloakID": "",
                  "loggedIn": false,
                  "profileID": "",
                  "registered": false,
                  "socialAccountsLinked": [],
                  "subscriptionFrequency": "",
                  "subscriptionLevel": "",
                  "userAgent": ""
                }
              }
            ]
          }
        ],
        "timeStamp": "2020-01-06T18:45:50.159Z",
        "processed": {
          "adobeAnalytics": false
        }
      }
    }
  ],
  "listing": {
    "browseFilter": "",
    "query": "",
    "queryMethod": "",
    "refinementType": "",
    "refinementValue": "",
    "resultCount": "",
    "searchType": ""
  },
  "page": {
    "attributes": {
      "queryParameters": ""
    },
    "category": {
      "primaryCategory": "products",
      "subCategories": [
        "rhel",
        "download"
      ]
    },
    "pageInfo": {
      "breadCrumbs": [
        "products",
        "rhel"
      ],
      "cms": "RHD CMS 8",
      "destinationURL": "https://developers.redhat.com/products/rhel/download",
      "errorMessage": "",
      "errorType": "",
      "language": "en",
      "pageID": "33925",
      "contentID": "33925",
      "pageName": "products",
      "referringDomain": "",
      "referringURL": "",
      "syndicationIds": [],
      "sysEnv": "desktop",
      "title": "Red Hat Enterprise Linux"
    },
    "listing": {
      "browseFilter": "",
      "query": "",
      "queryMethod": "",
      "refinementType": "",
      "refinementValue": "",
      "resultCount": "",
      "searchType": ""
    }
  },
  "product": {},
  "user": [
    {
      "profile": [
        {
          "profileInfo": {
            "accountID": "",
            "daysSinceLastPurchase": "",
            "daysSinceRegistration": "",
            "eloquaGUID": "271467ab-a3be-4c0e-b60b-905daa84c28c",
            "keyCloakID": "",
            "loggedIn": false,
            "profileID": "",
            "registered": false,
            "socialAccountsLinked": [],
            "subscriptionFrequency": "",
            "subscriptionLevel": "",
            "userAgent": ""
          }
        }
      ]
    }
  ]
}
```
